build-system/compile/compile-wrappers.js

```diff
@@ -71,7 +73,8 @@ exports.extension = function (
   // Use a numeric value instead of boolean. "m" stands for "module"
   const m = isModule ? 1 : 0;
   return (
-    `(self.AMP=self.AMP||[]).push({n:"${name}",${priority}${deps}` +
+    `(self.AMP=self.AMP||[]).push({n:"${name}",ev:"${version}",l:${latest},` +
```

Comment:
qq did we add these new fields to amp.extern.js?

Reply:
to @dvoytenko

Reply:
Thanks for the reminder. PTAL: https://github.com/ampproject/amphtml/pull/33098